sites/game-client/client.ejs

```diff
@@ -59,6 +59,13 @@ absolute URL.
       </div>
       <div id="modal-container" class="modal-container">
       </div>
+      <!-- context menu -->
```

Comment:
Optional: We can also consider using off the shelf library for the actual context menu.

Reply:
I believe that since the context menu has to be deeply integrated into our current design, it's quite difficult to implement it using off-the-shelf libraries. 
Given that the current solution works, I prefer to keep it as it currently is. However, if we later find it difficult to maintain the code, I'm open to using libraries.

Reply:
Ok, sure, let's leave the custom implementation as is.